### PR TITLE
app to compile .rho to .rbl file

### PR DESCRIPTION
**Pull request by dckc**
@pyrocto suggested: make it so the rholang compiler build file produces an app that takes a .rho file as input and produces a .rbl file instead of having to write stuff at the scala console and get stuff printed on stdout.

How does this look?

### CODE REVIEW

File: rholang/README.md [hunk 1]
```diff
@@ -4,18 +4,27 @@ Rholang is a behaviorally typed, concurrent programming language, with a focus o
...
-7. And then compile any Rholang ".rho" file with `Rholang2RosetteCompiler.main(Array("<path_to_Rholang_file>.rho"))`
+    * [BNFC](http://bnfc.digitalgrammars.com/) v2.8.1 Nov 2016 - e.g. `cabal install bnfc`
+3. Run `sbt bnfc:generate` to generate the parser
+4. Run `sbt compile` to compile classes
+5. Run `sbt assembly` to build a stand-alone .jar file
+
+## Command-line usage
+
+    $ java -jar target/scala-2.11/rholang-assembly-0.1-SNAPSHOT.jar examples/token.rho 
+    compiled examples/token.rho to examples/token.rbl
```
Comment (nashef) [inline rholang/README.md]: Can you wrap this in a shell script that passes its arguments to java?

File: rholang/src/main/scala/rholang/rosette/Compiler.scala [hunk 2]
```diff
@@ -78,7 +78,22 @@ object Rholang2RosetteCompiler extends RholangASTToTerm
   }
 
   def main(args: Array[String]): Unit = {
-    val result = compile(args(0))
-    println(serialize(result))
+    args.toList match {
```
Comment (nashef) [inline rholang/src/main/scala/rholang/rosette/Compiler.scala]: Why do you have to convert to List here? Does array not support the match/case syntax? Quick google search suggests it does.

File: rholang/README.md (hunk 1 above)
Comment (dckc) [reply rholang/README.md]: I think so.

File: rholang/src/main/scala/rholang/rosette/Compiler.scala (hunk 2 above)
Comment (dckc) [reply rholang/src/main/scala/rholang/rosette/Compiler.scala]: No good reason. Indeed, I see `case Array(_, "", _*)` in https://stackoverflow.com/questions/6647166/how-do-i-pattern-match-arrays-in-scala

My first attempt to write an array pattern didn't compile and using `toList` seemed quicker than looking it up.

I can fix this.

**Comment by dckc** [comment]
Would you rather I squashed 301303f into 4758195?

**Comment by KentShikama** [comment]
@dckc Thanks! Now if we could only get the boot-ess binary to execute on Ubuntu 14.04, play.rholang.org (equivalent of https://play.crystal-lang.org/#/cr) would be possible.

**Pull request merged**